staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1316,3 +1316,10 @@ type PartialObjectMetadataList struct {
 	// items contains each of the included items.
 	Items []PartialObjectMetadata `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// ObjectChangePersistedHeader is a header inserted in the response to
```

Comment:
The sentence sounds a little broken to me.
How about:
`ObjectChangePersistedHeader is a header inserted in the response to a modifying request, indicating whether ...`
Not sure about the , though.